mitodl/mit-open
# set default minimum score cutoff

### What are the relevant tickets?
closes https://github.com/mitodl/hq/issues/5659

### Description (What does it do?)
Adds a default minimum score cutoff to searches with a search term so we show fewer weakly relevant results

### How can this be tested?
Login as an admin and go to 

http://open.odl.local:8062/search

Verify that the minimum score cutoff slider in the Admin options facet of the search is set to 5

Make a search with a search term. Verify that the number of results increases as you set the minimum score to a lower value and decreases when you set it to a higher value

### Additional Context
I decided on 5 after playing with the setting on prod for a bit - we may adjust it higher or lower. 

https://github.com/mitodl/mit-open/pull/1590 was supposed to make the search setting settable from the environment. This worked for the backend but not the front end, so I need a commit to update these values for now. It probably makes sense to add a small api endpoint to communicate the values from the backend to the front end so we can set them from heroku but maybe that's not very high priority
